app/models/task.py

```diff
@@ -1,5 +1,37 @@
 from app import db
 
-
+#many tasks
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
+    goal = db.relationship("Goal", back_populates="tasks")
+
+    # Converts Task instance into a dict
+    def to_dict(self):
+        is_complete = False
+        if self.completed_at:
+            is_complete = True
+        dic_data = {
+            "id": self.task_id,
+            "title": self.title,
+            "description": self.description,
+            "is_complete": is_complete
```

Comment:
```suggestion
            "is_complete": True if completed_at else False
```
Using this syntax will allow you to conditionally set `is_complete` in a way that's a little bit cleaner than the if-else block.